sdmetrics/reports/single_table/diagnostic_report.py

```diff
@@ -49,31 +72,29 @@ def _get_num_iterations(self, property_name, metadata):
 
     def _print_results(self, out=sys.stdout):
```

Comment:
unrelated but do we need the `out` parameter? Is it ever set to anything besides sys.stdout?

Reply:
No that's a good point. Should I remove it?
Or modify the code to make the user able to use it (if they want to write the report output in a file for instance)

Reply:
Yeah I think you can just remove it

Reply:
Yes, done in [c277e80](https://github.com/sdv-dev/SDMetrics/pull/411/commits/c277e80072ca76014389bc3b1b12e6701d64747e).